src/sentry/rules/actions/sentry_apps/notify_event.py

```diff
@@ -146,3 +154,15 @@ def after(self, event: GroupEvent, state: EventState) -> Generator[CallbackFutur
             sentry_app=sentry_app,
             schema_defined_settings=self.get_option("settings"),
         )
+
+    def render_label(self) -> str | None:
+        sentry_app_installation_uuid = self._get_sentry_app_installation_uuid()
+
+        try:
+            sentry_app = SentryApp.objects.get(installations__uuid=sentry_app_installation_uuid)
```

Comment:
I could not use the existing `_get_sentry_app` function for this, as we need this function to not take in any other parameters, such that it works with how `render_label` is [used](https://github.com/getsentry/sentry/blob/44a1ce3eae79620e89a962d9a6ed1dd36ded4b48/src/sentry/api/serializers/models/rule.py#L27)